--- plugins/check_procs.orig   2016-05-10 15:24:27.000000000 +0200
+++ plugins/check_procs.c       2016-05-10 15:30:25.000000000 +0200
@@ -243,6 +243,8 @@
                                         printf("not considering - is myself or gone\n");
                                continue;
                        }
+                       /* Ignore child containers if we're on an OpenVZ hardware node */
+                       if (read_openvz_id(1) == 0 && read_openvz_id(procpid) != 0) continue;
                        /* Ignore parent*/
                        else if (myppid == procpid) {
                                if (verbose >= 3)
